src/Evaluators/common.jl

```diff
@@ -102,8 +102,8 @@ function _check(val, val_min, val_max)
     violated_sup = findall(val .> val_max)
     n_inf = length(violated_inf)
     n_sup = length(violated_sup)
-    err_inf = norm(val_min[violated_inf] .- val[violated_inf], Inf)
-    err_sup = norm(val[violated_sup] .- val_max[violated_sup] , Inf)
+    err_inf = xnorm_inf(val_min[violated_inf] .- val[violated_inf])
+    err_sup = xnorm_inf(val[violated_sup] .- val_max[violated_sup])
```

Comment:
This is really slow on the GPU. I guess `nrm2` won't work? If so, maybe one can do an optional argument.

Reply:
I agree. But as we are computing that only once, at the beginning, I think it won't be a bottleneck to use norm Inf here.